docs/RELEASE.md

```diff
@@ -0,0 +1,175 @@
+# Release process
+
+The release process for nobl9-go cannot be discussed without the context of
+our platform releases. In the following document, `n9` will refer to our
+platform **production** version.
+
+## The process
+
+1. Create release candidate version. \
```

Comment:
imho it would be good to highlight this line (and also the 2nd. point line so it will be visually easier to see that process consist of those two points,
you can use ### to do that and than treat description as a paragraph describing the point

Reply:
I think list is better suited here, if there would be longer descriptions and other nested lists inside I'd use new heading